contracts/PreERC721.sol

```diff
@@ -60,6 +60,8 @@ contract PreERC721 is ERC721, Initializable, Ownable, RoyaltiesReceiver {
 
     /// Overrides
     ///
+    /// @notice Returns the name of this ERC-721 Contract
+    /// @dev ¿I am not sure what _getImmutableVariablesOffset does?
```

Comment:
_getImmutableVariablesOffset is complex, but basically it's almost a hack inside EVM to get a location in memory where we stored things when we created this contract in a very cheap way